.github/workflows/ec-integration-tests.yml

```
@@ -0,0 +1,40 @@
+defaults:
+  run:
+    working-directory: test/erasure_coding
+
+jobs:
+  ec-integration-tests:
+    name: EC Integration Tests
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v4
```

actionlint (1.7.9)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)